JS/Airport.js

```diff
@@ -1,120 +1,83 @@
-const PassengerPlane = require('./Planes/PassengerPlane');
-const MilitaryPlane = require('./Planes/MilitaryPlane');
-const MilitaryType = require('./models/militaryType');
-const experimentalPlane = require('./Planes/experimentalPlane');
+const { PassengerPlane } = require('./planes/PassengerPlane');
+const { MilitaryPlane } = require('./planes/MilitaryPlane');
+const MilitaryType = require('./models/MilitaryType');
+const { ExperimentalPlane } = require('./planes/ExperimentalPlane');
+const { print } = require('./utils');
 
 class Airport {
-
-     getPasPl() {
-        let pl = this.planes;
-        var x = [];
-        for (let p of this.planes) {
-            if (p instanceof PassengerPlane) {x.push(p);}
+    
+     getPassengerPlanes() {
+        var passengerPlaneList = [];
+        for (let plane of this.planes) {
+            if (plane instanceof PassengerPlane) {passengerPlaneList.push(plane);}
         }
-        return x;
+        return passengerPlaneList;
     }
 
     getMilitaryPlanes() {
-        let militaryPlanes = [];
+        let militaryPlanesList = [];
         this.planes.forEach(plane => {
-            if (plane instanceof MilitaryPlane) {//if
-                militaryPlanes.push(plane);
+            if (plane instanceof MilitaryPlane) {
+                militaryPlanesList.push(plane);
             }
-            //else
-            else{}
         });
-//return
-        return militaryPlanes;
```

Comment:
Try to rewrite this code using `filter()` array method.

Reply:
Done